prefect_databricks/jobs.py

```diff
@@ -1148,7 +1148,7 @@ async def jobs_runs_get(
     | 401 | The request was unauthorized. |
     | 500 | The request was not handled correctly due to a server error. |
     """  # noqa
-    endpoint = "/2.1/jobs/runs/get"  # noqa
+    endpoint = "/2.0/jobs/runs/get"  # noqa
```

Comment:
Is it necessary to use the 2.0 jobs API? If not I suggest using the 2.1 jobs API to stay consistent with the other tasks in this module.
```suggestion
    endpoint = "/2.1/jobs/runs/get"  # noqa
```